packages/kernel-browser-runtime/src/PlatformServicesClient.test.ts

```diff
@@ -54,34 +54,37 @@ const makeNullReply = (messageId: `m${number}`): MessageEvent =>
     result: null,
   });
 
-describe('VatWorkerClient', () => {
+describe('PlatformServicesClient', () => {
   it('constructs with default logger', async () => {
     const stream = await TestDuplexStream.make(() => undefined);
     await stream.synchronize();
-    const client = new VatWorkerClient(
-      stream as unknown as VatWorkerClientStream,
+    const client = new PlatformServicesClient(
+      stream as unknown as PlatformServicesClientStream,
     );
     expect(client).toBeDefined();
   });
 
   it('constructs using static factory method', async () => {
-    const mockMessageTarget = makeMockMessageTarget();
-    const client = await VatWorkerClient.make(mockMessageTarget);
+    const client = await PlatformServicesClient.make({
+      postMessage: vi.fn(),
+      addEventListener: vi.fn(),
+      removeEventListener: vi.fn(),
+    } as unknown as PostMessageTarget);
```

Comment:
```suggestion
    const client = await PlatformServicesClient.make(makeMockMessageTarget());
```

this one should be reverted as well